slimta/queue/__init__.py

```diff
@@ -313,7 +315,7 @@ def enqueue(self, envelope):
         envelopes = self._run_policies(envelope)
         ids = self._pool_imap('store', self.store.write, envelopes,
                               repeat(now))
-        results = zip(envelopes, ids)
+        results = list(six.moves.zip(envelopes, ids))
```

Comment:
After another look, since we added the `list()`, I don't think we need to use `six.moves.zip` at all. It seems to me like the behavior of this would be the same on py2 and py3: `list(zip(envelope, ids))`.